APE14.rst

```diff
@@ -144,6 +144,20 @@ low-level API recommends:
             The number of axes in the world coordinate system
             """
 
+        @property
+        def shape_pixel(self):
+            """
+            The shape of the data that the WCS applies to.
```

Comment:
Maybe add ``(optional)`` or something at the end of this line?  I suspect many will read the lead sentence but not the paragraph...

Also, although it's pretty strongly implied by being a ``shape*``, perhaps say something like "a tuple of length n_wcs"?

Reply:
Done!